Formula/git-gui.rb

```diff
@@ -0,0 +1,49 @@
+class GitGui < Formula
+  desc "Tcl/Tk UI for the git revision control system"
+  homepage "#{Formula["git"].homepage}"
+  url "#{Formula["git"].stable.url}"
+  sha256 "#{Formula["git"].stable.checksum}"
+  head "#{Formula["git"].head}", :shallow => false
+
+  depends_on "git"
+  depends_on "tcl-tk"
+
+  def install
+    ENV["V"] = "1" # build verbosely
+    args = %W[
+      prefix=#{prefix}
+      gitexecdir=#{libexec}/git-core
+      sysconfdir=#{etc}
+      CC=#{ENV.cc}
+      CFLAGS=#{ENV.cflags}
+      LDFLAGS=#{ENV.ldflags}
+    ]
+
+    # By setting TKFRAMEWORK to a non-existent directory we ensure that
+    # the git makefiles don't install a .app for git-gui
+    # We also tell git to use the homebrew-installed wish binary from tcl-tk.
+    # See https://github.com/Homebrew/homebrew-core/issues/36390
+    tcl_bin = Formula["tcl-tk"].opt_bin
+    args += %W[
+      TKFRAMEWORK=/we/do/not/want/to/install/a/.app/for/git-gui
+      TCL_PATH=#{tcl_bin}/tclsh
+      TCLTK_PATH=#{tcl_bin}/wish
+    ]
+    system "make", "-C", "git-gui", "install", *args
+    system "make", "-C", "gitk-git", "install", *args
```

Comment:
Do these install a single binary (each) or more than that? If a single binary maybe it's just worth doing `bin.install` instead?

Reply:
They install multiple files each:
```
/usr/local/Cellar/git-gui/2.25.0
/usr/local/Cellar/git-gui/2.25.0/bin
/usr/local/Cellar/git-gui/2.25.0/bin/gitk
/usr/local/Cellar/git-gui/2.25.0/libexec
/usr/local/Cellar/git-gui/2.25.0/libexec/git-core
/usr/local/Cellar/git-gui/2.25.0/libexec/git-core/git-gui
/usr/local/Cellar/git-gui/2.25.0/libexec/git-core/git-citool
/usr/local/Cellar/git-gui/2.25.0/libexec/git-core/git-gui--askpass
/usr/local/Cellar/git-gui/2.25.0/share
/usr/local/Cellar/git-gui/2.25.0/share/git-gui
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/choose_font.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/branch_delete.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/remote_add.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/status_bar.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/browser.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/line.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/about.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/spellcheck.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/git-gui.ico
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/chord.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/win32.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/sshkey.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/mergetool.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/error.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/win32_shortcut.js
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/console.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/logo.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/date.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/transport.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/branch.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/merge.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/branch_checkout.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/encoding.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/branch_rename.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/shortcut.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/commit.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/remote.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/search.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/tclIndex
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/el.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/sv.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/ja.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/pt_br.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/ru.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/fr.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/pt_pt.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/vi.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/nb.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/bg.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/hu.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/zh_cn.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/de.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/msgs/it.msg
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/tools.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/database.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/checkout_op.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/choose_repository.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/blame.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/option.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/diff.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/index.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/remote_branch_delete.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/branch_create.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/choose_rev.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/themed.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/tools_dlg.tcl
/usr/local/Cellar/git-gui/2.25.0/share/git-gui/lib/class.tcl
/usr/local/Cellar/git-gui/2.25.0/share/gitk
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/sv.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/ja.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/pt_br.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/ru.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/fr.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/pt_pt.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/vi.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/ca.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/bg.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/es.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/hu.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/zh_cn.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/de.msg
/usr/local/Cellar/git-gui/2.25.0/share/gitk/lib/msgs/it.msg
```